tests/vcf_test.py

```diff
@@ -27,11 +27,13 @@ def genome_reader() -> ReferenceGenome:
 
 
 class TestVCF:
-    def test_header(self, genome_reader):
+    def test_header_no_data(self, blocks, genome_reader):
```

Comment:
🔧 do we need the `blocks` fixture here? I don't think its used. Maybe `test_header_only()` might be a clearer name?